astrolabe/utils.py

```diff
@@ -251,8 +251,9 @@ def stop(self):
 
         # Workload executors wrapped in shell scripts can report that they've
         # terminated earlier than they actually terminate on Windows.
+        # One of the reasons for this that sometimes we need to write more than N milliones log lines to the file
         if self.is_windows:
-            sleep(2)
+            sleep(60)
```

Comment:
In most cases, it's enough to have much less timeout here, like 5-10 seconds, but very rarely even 30 seconds was not enough, so I set the safest (for now) value

Reply:
Can you please transfer this note into the source code as a comment so that developers looking at it in the future have the context for how this value was selected?

Reply:
done